native/dgetrf.go

```diff
@@ -29,7 +29,7 @@ func (impl Implementation) Dgetrf(m, n int, a []float64, lda int, ipiv []int) (o
 		panic(badIpiv)
 	}
 	if m == 0 || n == 0 {
-		return
+		return false
```

Comment:
Should this return false? Lapack returns the equivalent of true. I hate these "empty set" issues.